src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+export const CHECK_STATUS = 'CHECK_STATUS';
+
+export const initialState = {
+  status: '',
+};
+
+export default function categoriesReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return {
+        ...state,
+        status: action.status,
+      };
```

Comment:
- [ ] Please return a string "Under construction" For the check status action as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/178458075-5a69f04a-10b0-43af-bc87-1dc21f8d38d0.png)